src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -53,9 +53,9 @@ public abstract class FastTreeTrainerBase<TArgs, TTransformer, TModel> :
         where TArgs : TreeArgs, new()
         where TModel : class
     {
-        protected readonly TArgs Args;
-        protected readonly bool AllowGC;
-        protected int FeatureCount;
+        internal readonly TArgs Args;
```

Comment:
>internal [](start = 8, length = 8)

Any reason why you change protected to internal? Protected visible only for derived classes, they not visible to user anyway.

Reply:
In other PRs you tend to change `protected` to `private protected` why not here?

---
In reply to: [256066643](https://github.com/dotnet/machinelearning/pull/2511#discussion_r256066643) [](ancestors = 256066643)